channels/packages/cli/2.9.0/deployment-storageos-cli.yaml

```diff
@@ -38,7 +38,7 @@ spec:
                   optional: false
             - name: STORAGEOS_ENDPOINTS
               value: storageos:5705
-          image: storageos/cli
+          image: cli
```

Comment:
Is this intended?

Reply:
Yes, it's overwritten by the cli controller. The same way we use `controller` or `manager` as placeholders for other images